python-threatexchange/threatexchange/cli/tests/e2e_test_helper.py

```diff
@@ -70,3 +70,13 @@ def assert_cli_usage_error(
         self.assertEqual(exception.returncode, 2)
         if msg_regex is not None:
             self.assertRegex(str(exception), msg_regex)
+
+    def assert_cli_error_output(
+        self, given_args: t.Iterable[str], expected_err: t.Union[str, t.Dict[int, str]]
+    ) -> None:
```

Comment:
ignorable: Consider making cli_usage_error a subset of this behavior - searching by error regex is slightly more flexible than searching by exact string.